src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -158,17 +155,14 @@ for live streaming and video on demand for the web.
 You can use the same method to create a key for both DASH and HLS. Do this using
 openssl. The following will create a key made of 16 hex values.
  
-    openssl rand -out media.key 16
+    openssl rand -hex -out media.key 16
```

Comment:
key_id should be randomly generated as well. Why not output the result to console instead of saving to file then copying out?

Reply:
key_id shouldn't be a copy of the media key.

Reply:
That's not my understanding. Did it change? If so, when?